source/analysis/AnalysisManager.cpp

```diff
@@ -236,6 +236,46 @@ void AnalysisManager::getFunctionDrivers(const CallExpression& expr, const Symbo
         getFunctionDrivers(*call, containingSymbol, visited, drivers);
 }
 
+void AnalysisManager::getTaskTimingControls(const CallExpression& expr,
```

Comment:
I think this is different enough from getFunctionDrivers that it's ok to just have it separate, even though at first glance there is some overlap.